src/react-components/home/SignInButton.scss

```diff
@@ -0,0 +1,15 @@
+@use "../styles/theme.scss";
+
+:local(.sign-in-button) {
+    display: none;
```

Comment:
I want to see why this is display none over here.

Reply:
I'm not 100% sold on this solution but the idea being 'sign-in-button' is the version of the button that has `display:none` for mobile breakpoints since it shouldn't exist in the mobile header/nav. Open to other solutions and suggestions. 
https://www.figma.com/file/J6YKkb2vWDlk81zm4kxUzA/Hubs-discoverability-and-onboarding?node-id=375%3A0

Reply:
I think it's fine. Thanks for looking at it. I alternatively might have written it into the react component and removed it when the screen was x size, but the css solution is cleaner.